src/component/feature/FeatureToggleList/FeatureToggleListTable/FeatureToggleListTable.tsx

```diff
@@ -141,6 +155,24 @@ export const FeatureToggleListTable: VFC<IExperimentProps> = ({
         }
     }, [setHiddenColumns, isSmallScreen, isMediumScreen]);
 
+    useEffect(() => {
+        const tableState: IPageQuery = {};
+        if (sortBy[0]?.id !== defaultSort.id) {
```

Comment:
These checks could be clearer. What does `sortBy[0]` refer to? Is it the header field?

Reply:
Format from `react-table`. Array, because it can support multi-sort (that we don't use)